service/src/main/resources/application.yml

```diff
@@ -16,6 +16,7 @@ spring:
     throw-exception-if-no-handler-found: false
   resources:
     add-mappings: false
+  cloud.bus.enabled: false
```

Comment:
LGTM, although nit: I'd stick with the yaml flow for defining properties:
```
  cloud:
    bus:
      enabled: false
```
Just for consistency.